scopes/lanes/lanes/switch.cmd.ts

```diff
@@ -5,34 +5,34 @@ import { MergeStrategy } from '@teambit/legacy/dist/consumer/versions-ops/merge-
 import { LanesMain } from './lanes.main.runtime';
 
 export class SwitchCmd implements Command {
-  name = 'switch <lane> [pattern]';
+  name = 'switch <lane> [pattern]'; // @david why do we have the pattern flag here?
   description = `switch to the specified lane`;
   extendedDescription = ``;
   private = true;
   alias = '';
   arguments = [
     {
       name: 'lane',
-      description: 'lane-name or lane-id (if not exists locally) to switch to',
+      description: 'lane-name or lane-id (if lane is not local) to switch to',
     },
   ];
   options = [
     [
       'n',
       'alias <string>',
-      'relevant when the specified lane is a remote late. name a local lane differently than the remote lane',
+      'relevant when the specified lane is a remote late. name a local lane differently to the remote lane',
```

Comment:
I have a typo here :) it should be `specified lane is a remote lane.` (not "late").